### PR TITLE
build duckdb and python-duckdb packages

### PR DESCRIPTION
**Pull request by ytausch**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

We now build `duckdb` next to the `python-duckdb` package because `duckdb` once was a duplicate of `python-duckdb`.

See https://github.com/conda-forge/duckdb-feedstock/issues/6

@xhochy Does this work like this or do we have to whitelist this feedstock so it can build the `duckdb` package?

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,46 +12,61 @@ source:
...
+  - name: python-{{ name|lower }}
+    script: build-duckdb.sh  # [unix]
+    script: build-duckdb.bat  # [win]
+    requirements:
+      build:
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - pybind11 >=2.6                         # [build_platform != target_platform]
+        - setuptools_scm <7
+        - {{ compiler('c') }}
```
Comment (xhochy) [inline recipe/meta.yaml:29]:
```suggestion
        - {{ stdlib('c') }}
        - {{ compiler('c') }}
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,46 +12,61 @@ source:
...
+        - {{ compiler('cxx') }}
+      host:
+        - pybind11 >=2.6
+        - python
+        - pip
+        - setuptools_scm <7
+      run:
+        - python
+        - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx and x86_64]
```
Comment (xhochy) [inline recipe/meta.yaml:38]:
```suggestion
```

**Comment by xhochy** [comment]
To make this work, you need to delete `recipe/conda_build_config.yaml`

**Comment by ytausch** [comment]
@xhochy Do you know what fails now? The error is sadly not very descriptive

File: recipe/meta.yaml [hunk 3]
```diff
@@ -12,46 +12,61 @@ source:
...
+        # ppc64le/aarch64 is emulated and fails to run the test suite here
+        # Skip slow tests to avoid memory issues in CI, skip stub tests that is fragile under new version of mypy
+        - pytest tools/pythonpkg/tests --ignore=tools/pythonpkg/tests/slow/ --ignore=tools/pythonpkg/tests/stubs/  # [not (ppc64le or aarch64)]
+  - name: {{ name|lower }}
+    build:
+      noarch: generic
+      skip: true  # [not linux64]
+      requirements:
+        run:
+          - {{ pin_subpackage('python-duckdb', max_pin="x.x.x") }}
```
Comment (xhochy) [inline recipe/meta.yaml:65]:
```suggestion
          - python-duckdb {{ version }}
```

**Comment by ytausch** [comment]
@xhochy This doesn't seem to have helped - and also, I saw `pin_subpackage` in [various places](https://github.com/conda-forge/py-rattler-feedstock/blob/main/recipe/meta.yaml) for aliased package outputs.

**Comment by xhochy** [comment]
> @xhochy This doesn't seem to have helped - and also, I saw `pin_subpackage` in [various places](https://github.com/conda-forge/py-rattler-feedstock/blob/main/recipe/meta.yaml) for aliased package outputs.

Yes, then it's best probably to revert here

File: recipe/meta.yaml [hunk 4]
```diff
@@ -12,46 +12,61 @@ source:
...
+      commands:
+        - pip check
+        # ppc64le/aarch64 is emulated and fails to run the test suite here
+        # Skip slow tests to avoid memory issues in CI, skip stub tests that is fragile under new version of mypy
+        - pytest tools/pythonpkg/tests --ignore=tools/pythonpkg/tests/slow/ --ignore=tools/pythonpkg/tests/stubs/  # [not (ppc64le or aarch64)]
+  - name: {{ name|lower }}
+    build:
+      noarch: generic
+      skip: true  # [not linux64]
+      requirements:
```
Comment (xhochy) [inline recipe/meta.yaml:63]: @ytausch This is the actual issue 😂 
```suggestion
    requirements:
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 4 above)
Comment (ytausch) [reply recipe/meta.yaml]: Oh no, yes xD

**Review by xhochy — APPROVED** [review]

**Pull request merged**